addon/components/ember-sparkles/pie-chart.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import { select } from 'd3-selection';
+import layout from '../../templates/components/ember-sparkles/pie-chart';
+
+export default Ember.Component.extend({
+  layout,
+  tagName: 'g',
+  classNames: ['ember-sparkles--pie-chart'],
+  'with-transition': true,
+  'with-arc-labels': false,
+
+  didInsertElement() {
+    this._super(...arguments);
+    Ember.run.scheduleOnce('afterRender', this, 'renderChart');
```

Comment:
Like this:
https://github.com/LocusEnergy/ember-sparkles/blob/area-helper/addon/components/ember-sparkles/d3-element.js

used like: https://github.com/LocusEnergy/ember-sparkles/blob/area-helper/addon/components/ember-sparkles/bar-chart.js